index.js

```diff
@@ -17,8 +17,24 @@ const EMAIL_PERMISSION = "alexa::profile:email:read";
 const EMPTY_STATE = 'empty';
 // used to move on to next step
 const TERMINATION_WORD = 'next';
+const helpWords = ['help'];
```

Comment:
If it is only one word we do not need to put it in an array we can make it stand-alone.